docs/en/users/05_Configuration.md

```diff
@@ -51,15 +88,53 @@ The duration of this notification can be set. By default, the value is 0.
 
 # Sharing
 
-**TODO**
+To make your life easier, you can share directly an article within FreshRSS.
+
+At the moment, FreshRSS supports 15 sharing methods ranging from self-hosted services (Shaarli, etc.) to proprietary services (Facebook, etc.).
+
+By default, the sharing list is empty.
+![Sharing configuration](../img/users/configuration.sharing.png)
+
+To add a new item in the list, follow those simple steps:
+
+ 1. Select the share method in the drop-down.
+ 1. Press the ```✚``` sign to add it to the list.
+ 1. Configure the method in the list. All method names can be modified in the display. Some methods need the sharing URL to be able to work properly (ex: Shaarli).
+ 1. Submit your changes.
+
+To remove an item from the list, follow those simple steps:
+
+ 1. Press the ```❌``` sign next to the share method you want to remove.
+ 1. Submit your changes.
 
 # Shortcuts
 
-**TODO**
+To ease the use of the application, FreshRSS comes with a lot of predefined keyboard shortcuts.
+They allow a lot of actions to improve the user experience with a keyboard.
```

Comment:
Maybe `They offer many actions`? (`comes with a lot` immediately followed by `allow for a lot`)